### DISCUSSION TITLE
How to update,

### DISCUSSION THREAD

**Discussion by gembrian1983** [sct/overseerr#2661] [General]
Hi All, fairly new user, i am being told that version is out of date, i see a new version is available. could someone advise me how to update as i cannot se the option? thanks.

**Comment by TheCatLady** [comment]
How did you install Overseerr?

**Comment by tommitchell00** [comment]
I am looking for this also, as for linux, the app is no longer supported through snap and that is how I installed it.

**Reply by cevadau** [reply]
I am in the same situation

**Reply by TheCatLady** [reply]
Snap is still supported, there just aren't going to be new builds until the upstream issue is fixed.

**Reply by slconrad** [reply]
Greetings, do you have any estimate as to when upstream is fixed? I can update on RPi (snap) until this is resolved. Appreciate  the effort that goes into this app!

**Reply by TheCatLady** [reply]
Unfortunately we have no control over this, so there is no timeline for a fix.

**Reply by slconrad** [reply]
Greeting! Is there any update on the upstream fix on Snap? I would love to try out the new features and I understand you don't have control over this, but hopefully you can get feedback from the Snap dev team on this issue. Thanks again!

**Comment by DevMan01** [comment]
I think we all understand that `snap` is a blocker that really is outside the core dev's ( @TheCatLady ) control. In fact Overseerr isn't the only app hit by this, Tautulli was impacted by this not too long ago as well. 

Until a solution is available, is there a workaround we can try, that preserves the data of our install, while adding new codebase and features?

**Reply by DevMan01** [reply]
Hey @sct - bumping this up, wondering if there is any update, insight, or workarounds you can offer?

**Comment by azazael13** [comment]
Is there an update to this, or plans to use something other than Snap for Ubuntu?

**Reply by TheCatLady** [reply]
The snap build workflow is now working, and snap builds will be available for the next stable release.

**Reply by azazael13** [reply]
Good to hear, thanks.

**Reply by DevMan01** [reply]
Very cool! Awesome to hear.

**Reply by SiskoUrso** [reply]
Great news! Thank you for the update.

**Comment by tommitchell00** [comment]
That's great, 
Thank you
